footer.css

```diff
@@ -0,0 +1,353 @@
+.footer-content {
+  margin: auto;
+  width: 100%;
+  max-width: 1200px;
+}
+
+.footer-content section {
+  display: inline-flex;
+}
+
+.footer-content div {
+  display: inline-flex;
+}
+
+.text-before-footer {
+  margin: auto;
+  width: 100%;
+  max-width: 1200px;
+}
+
+.text-before-footer div {
+  display: flex;
+}
+
+.first-text-content {
+  margin-top: 3.5%;
+  width: 100%;
+  padding-bottom: 12px;
+  padding-top: 15px;
+  max-width: 840px;
+  border-top: 1px black solid;
+}
+
+.txt-1 {
+  font-family: franklin-normal-700, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #333;
+}
+
+.txt-1 a {
+  text-decoration: none;
+  font-family: franklin-normal-700, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #333;
+}
+
+.external-links a {
+  text-decoration: none;
+  color: #666;
+  font-size: 11px;
+  font-family: franklin-normal-500, Arial, sans-serif;
+}
+
+.sub-categories a {
+  font-family: franklin-normal-500, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  color: black;
+}
+
+.txt-5 a {
+  margin: 15px auto auto 900px;
+  font-family: franklin-normal-500, Arial, sans-serif;
+  font-size: 11px;
```

Comment:
You have used same `font-family` property to many classes. Instead of that you can make a separate class for font and use that class wherever needed in HTML. It makes your code cleaner.
**OR**

you can do like this
```
.external-links a,
.sub-categories a,
.txt-5 a,
.link3 a,
.link2 a {
  font-family: franklin-normal-700, Arial, sans-serif;
}
```

Do not repeat the code follow the DRY principle.